.github/workflows/test-indexer-debian.yml

```diff
@@ -0,0 +1,42 @@
+name: Test Wazuh indexer on upgrade - Debian
+on:
+  push:
+    branches:
+      - 1850-wazuh-indexer-preserve-config-files-upon-upgrade-master
+  pull_request:
+    paths:
+      - 'stack/indexer/deb/debian/postrm'
+      - 'stack/indexer/deb/debian/postinst'
+      - 'stack/indexer/deb/debian/preinst'
+  workflow_dispatch:
+
+jobs:
+  Test-debian-indexer-upgrade:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download wazuh-indexer release package
```

Comment:
If we refer to a package, the name of the package (`Wazuh indexer`) should be used, if it is a service, the name of the service (`wazuh-indexer`)

Reply:
Indeed. Done.